server/src/config/routes.ts

```diff
@@ -0,0 +1,184 @@
+/* tslint:disable */
```

Comment:
this file is not needed after migrating server to nest

Reply:
remove routes.ts